x-pack/plugins/maps/public/connected_components/widget_overlay/layer_control/layer_toc/toc_entry/toc_entry_actions_popover/toc_entry_actions_popover.tsx

```diff
@@ -43,6 +66,22 @@ export class LayerTocActions extends Component {
     }));
   };
 
+  _cloneLayer = () => {
```

Comment:
these are methods on the prototype, and don't need to be recreated for each instance. use `_cloneLayer(){...}`. They're also already closed in the `onClick` lambda which preserves the `this` context.